scripts/run_frontend_tests.sh

```diff
@@ -37,28 +37,60 @@ set -e
 source $(dirname $0)/setup.sh || exit 1
 source $(dirname $0)/setup_gae.sh || exit 1
 
-# Install third party dependencies
-# TODO(sll): Make this work with fewer third-party dependencies.
-bash scripts/install_third_party.sh
-
-# Ensure that generated JS and CSS files are in place before running the tests.
-echo ""
-echo "  Running build task with concatenation only "
-echo ""
+# Parse additional command line arguments.
```

Comment:
We use a similar pattern in run_e2e_tests.sh. Can you abstract both instances out into a common function (maybe in setup.sh) to avoid duplication?